ee/management/commands/create_kafka_topics.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from django.core.management.base import BaseCommand
+from kafka import KafkaAdminClient
+from kafka.errors import TopicAlreadyExistsError
+
+from ee.kafka_client.client import _KafkaSecurityProtocol
+from ee.kafka_client.helper import get_kafka_brokers, get_kafka_ssl_context
+from ee.kafka_client.topic_definitions import TopicDefinition
+from ee.kafka_client.topics import KAFKA_TOPIC_DEFINITIONS
+from posthog.settings.data_stores import KAFKA_SECURITY_PROTOCOL
+
+
+def create_kafka_topics(kafka_admin_client: KafkaAdminClient, topic_definitions: List[TopicDefinition]):
+    topics = [topic_definition.get_new_topic_definition() for topic_definition in topic_definitions]
+    created_topics: List[str] = []
+    existing_topics: List[str] = []
+    missing_topics: List[str] = []
+    for topic in topics:
+        try:
+            kafka_admin_client.create_topics([topic])
+            created_topics.append(topic.name)
+        except TopicAlreadyExistsError:
+            print(f"Topic {topic.name} already exists. Skipping creation...\n")
+            existing_topics.append(topic.name)
+        except Exception as e:
+            print(f"Could not create topic {topic.name} with error: {e.__str__()}")
```

Comment:
Use logger instead of print, passing the error as a separate arg. Decent logging platforms will be able to take advantage of that.